src/configeditor/hooks.ts

```diff
@@ -117,13 +121,79 @@ function useUpdateConfig() {
   };
 }
 
+function validateBoundingBoxes(type: string, bounds: BoundingBox | null, errors: ValidationErrors) {
+  if (typeof errors.boundingBoxes === 'string') return;
```

Comment:
I try it out, and there are some problems:
- If I enter a nonsense value like "xx", it complains about "empty field is not allowed"
- Entering -3 in east and 1 in west does not give an error
- There should be a check that either all fields are filled in, or none of them

These validation functions are perfect candidates for unit tests! (<- mostly reminder to myself, but feel free)